includes/DefaultSettings.php

```diff
@@ -252,6 +259,17 @@
 	require_once( $wgLocalSettingsFile );
 }
 
+//If http host is not in whitelist reset config to original request host
+if (!in_array($_SERVER['HTTP_HOST'], $wgRemoteAddrWhitelistedHosts )) {
+    $_SERVER["HTTP_HOST"] = $origHttpHost;
```

Comment:
maybe change to single quote here too, for consistency

Reply:
probably copy pasted it before I changed it :-)